superset-frontend/packages/superset-ui-chart-controls/test/utils/getColorFormatters.test.ts

```diff
@@ -133,6 +195,21 @@ describe('getColorFunction()', () => {
     expect(colorFunction(100)).toEqual('rgba(255,0,0,1)');
   });
 
+  // it('getColorFunction EQUAL inverse', () => {
```

Comment:
Remove?

Reply:
I left it in so I'd remember to ask...Is there any reason to try to make this work with the equals operator? I haven't really used the conditional formatting feature much, but it looks like equals will only color cells that are exactly equal, and at 100% opacity, so there's no need to be able to inverse it. Right?

Reply:
@kgabryje what do you think?

Reply:
I think it's a good idea to ask @kasiazjc to review the screen layout with the new checkbox. Is the proposed solution ok or do we need to change the layout?

Reply:
As for the available colours, those were selected for feature parity with a heatmapping functionality that existed before.
I like the idea of allowing picking more colors/typing hex! What would work even better is somehow adding a color picker to the modal instead of letting user manually type hex

Reply:
I'm not sure if highlighting the checkbox in red is a good UX in this case. Could we just disable the checkbox and add a tooltip with explanation?

Reply:
These are the existing validation errors
![between_error_1](https://user-images.githubusercontent.com/70416691/173390982-91dc2ae6-18c7-4af5-bf53-063c3e0d0f1a.png)
![between_error_2](https://user-images.githubusercontent.com/70416691/173390994-8bbb188c-b9e3-45fc-834f-919ae8e9ba6e.png)

This is what the inverse validation does (doesn't actually highlight checkbox in red, just gives the error)
![inverse_error](https://user-images.githubusercontent.com/70416691/173391009-8e430712-3234-423c-8bc9-059de7b03592.png)

I don't know how to disable it. Disabled isn't a valid property on this component, and I can't find any other superset code that uses this Checkbox component (why is that?). Looks like control panels use the CheckboxControl from src/explore/components/controls/CheckboxControl, but that's a jsx file so I figured the Checkbox.tsx was the one you're moving toward...?

Reply:
And what do you think of this instead?
![base_color](https://user-images.githubusercontent.com/70416691/173423221-ae44d6fb-7111-4a19-801c-a9bbbab2f79a.png)

Date picker, and it respects the alpha chosen as the max alpha. Pretty sure the date picker broke all of the tests though, so I'll have to fix that to push